source/nodejs/adaptivecards/src/host-config.ts

```diff
@@ -358,6 +367,14 @@ export class HostConfig {
                 extraLarge: obj.fontSizes && obj.fontSizes["extraLarge"] || this.fontSizes.extraLarge
             };
 
+            this.lineHeights = {
```

Comment:
How about adding an `if (obj.lineHeights)` statement around this instead of repeating it 5 times? I know the same pattern is used in other places, but it's never too late to start :)

Reply:
done